### ISSUE TITLE
Recommended For You is not loading Odysee Premium badges

### ISSUE THREAD

**Issue by Abwasserrohr** [OdyseeTeam/odysee-frontend#1169]
**Bug**
Recommended For You is not loading Odysee Premium badges

**To Reproduce**

1. Go to odysee.com with a Premium account to access the `Recommended For You` feature.
2. Wait until a channel appears that should have a badge.
3. Channel has no badge, it only has a badge if it's already loaded from somewhere else.

**Expected behavior**
Recommended should load the badges for channels too.

**Video**

https://user-images.githubusercontent.com/44294954/159303712-88892ebb-cba3-40f6-9d77-5013c99efb25.mp4

**System Configuration**
Chrome on Windows

**Comment by infinite-persistence** [comment]
Yup, either missed or excessive https://github.com/OdyseeTeam/odysee-frontend/issues/1136#issuecomment-1074048418
Issue should go away if we just shadow `resolve`.

**Comment by tzarebczan** [comment]
This is still an issue, along with comment replies.

**Comment by tzarebczan** [comment]
Not loading in search results either.
